index.html

```diff
@@ -26,18 +26,43 @@
             </a>
         </nav>
     </header>
-    <div class="container">
-        <main>
+    <div class="container flex-container">
+        <main class="flex-container">
             <section>
                 <div class="name">
                     <h1>Felix Odete</h1>
                 </div>
-                <div class="about-me">
+                <div class="about-me flex-container">
                     <p>Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
                         If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
                 </div>
-                <div class="rectangle-button">
-                    <h1>Start collaboration</h1>
+                <a href="">
+                    <div class="rectangle-button">
+                        <h1>Start collaboration</h1>
+                    </div>
+                </a>
+                <div class="middle-image"></div>
+            </section>
+            <section>
+                <h2 class="projects">Projects</h2>
+            </section>
+            <section class="computer-image"><h2></h2></section>
+            <section class="project-section">
+                <h2>Project name goes here</h2>
+            </section>
+            <section class="grid-container">
+                <div class="html-css-rectangle">
+                    <p>HTML/CSS</p>
+                </div>
```

Comment:
- [ ] Please all the work sections suppose to be in one single section tag if you are only using the element as a styling wrapper, use a div

![image](https://user-images.githubusercontent.com/74744787/158980358-c47bc49b-4083-4141-977f-13e44e7d2dc3.png)

Reply:
Changed the code to have this in one section tag.